web/test/pages/profile/profile-foreign.test.tsx

```diff
@@ -449,4 +455,140 @@ describe("profile-foreign", () => {
       });
     });
   });
+
+  describe("profile opportunities alert for Dakota Nexus", () => {
+    const ProfileConfig = Config.profileDefaults.fields;
+
+    it("displays alert on info tab", () => {
+      const business = generateBusinessForProfile({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING",
+          businessPersona: "FOREIGN",
+          ...getProfileDataMixin("FOREIGN"),
+          municipality: undefined,
+          dateOfFormation: undefined,
+        }),
+      });
+      renderPage({ business });
+      expect(screen.getByTestId("opp-alert")).toBeInTheDocument();
+    });
+
+    it("lists each unanswered funding/certification question", () => {
+      const business = generateBusinessForProfile({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING",
+          businessPersona: "FOREIGN",
+          ...getProfileDataMixin("FOREIGN"),
```

Comment:
[pebble] I've never seen the word "mixin" in code outside of the Sass framework, so this one made me have to go look up the implementation to understand what was going on. Maybe we could name it something more descriptive like `getForeignNexusProfileFields`

Reply:
Renamed